liblll/Compiler.h

```diff
@@ -30,9 +30,11 @@ namespace dev
 namespace eth
 {
 
+using ReadCallback = std::function<std::string(std::string const&)>;
+
 std::string parseLLL(std::string const& _src);
-std::string compileLLLToAsm(std::string const& _src, bool _opt = true, std::vector<std::string>* _errors = nullptr);
-bytes compileLLL(std::string const& _src, bool _opt = true, std::vector<std::string>* _errors = nullptr);
+std::string compileLLLToAsm(std::string const& _src, bool _opt = true, std::vector<std::string>* _errors = nullptr, ReadCallback const& _readFile = ReadCallback());
```

Comment:
The shorter way: `ReadCallback const& _readFile = {}`.

Reply:
I'd prefer this way because that's how the other places do it. Probably useful to create a cleanup PR and decide which pattern is better (I don't mind which).